docs/running-frontend-against-examples.md

```diff
@@ -0,0 +1,23 @@
+# Running your frontend against the examples
+
+The dummy content store allows you to run a frontend using the examples in this repo.
+
+It has the same API as the real [content-store](https://github.com/alphagov/content-store.)
+
+You will not need to run [content-store](https://github.com/alphagov/content-store] itself.
+
+From this repo, run:
+
+```
+dummy_content_store
```

Comment:
Maybe write this as:

```
$ bundle exec dummy_content_store
```

- the `$` prefix is an idiomatic way of indicating that this is something that you type into a command line shell
- I think it's helpful to remind people to run `bundle exec` as well, although it may work without (depending on your setup), using `bundle exec` will always work